docs/source/markdown/options/add-host.md

```diff
@@ -27,3 +28,8 @@ You can prevent that by either giving the **--no-hosts** option, or by setting
 address was configured manually and Podman fails to determine the IP address
 automatically, Podman will silently skip adding these internal hostnames to
 `/etc/hosts`.
+
+Podman will use the host's `/etc/hosts` file as a basis by default, i.e. any
+hostname present in this file will also be present in the container's
```

Comment:
I'll have to push again due to the discussion in https://github.com/containers/common/pull/2184. However, I'm not entirely sure what you're suggesting. You mean like "the `/etc/hosts` file of the host" instead of "the host's `/etc/hosts` file"? Because especially "the host's" and "the container's" is used all over the docs and the "of the host" appendix feels a bit cumbersome to me as non-native speaker :see_no_evil: I'll rephrase it if that's wished though.

Reply:
> You mean like "the `/etc/hosts` file of the host" instead of "the host's `/etc/hosts` file"?

Correct.